data/free worlds start.txt

```diff
@@ -1399,6 +1399,162 @@ mission "FW Pirates 3.1B"
 
 
 
+mission "FW Pirates 3.2C"
+	name "Crush the Pirates of Greenrock"
+   	description "Destroy the combined pirate armada while they're concentrated in this system, then put an end to their way of life on Greenrock."
+	blocked <message>
+	deadline 0
+	deadline 1
+	priority
+	landing
+	clearance "There's no response from Greenrock's Traffic Control, and their systems aren't active. Perhaps the people there are part of the pirate armada's crew? On the planet at least, there's nothing left to stop you from landing." 
+	infiltrating
+	to offer
+        has "FW Pirates 3.2A: done"
+        "combat rating" > 5000
+	source Greenrock
+	destination Greenrock
+	on offer
+        conversation
+		`As you bring your ship in for a landing on <planet>'s main spaceport, you are struck by how little this fight meant.  You have made a dent in their local military strength, but below you can see that the pirate shipyards, outfitters, fuel depots, markets, and other support infrastructure remain, supported by their slave laborers. You have serious doubts about Tomek's plan to annex this world of determined anarchists, slavers, and pirates. As it stands, this world soon will be back to business as usual with new ships and fresh crews.`
+		`    An idea forms in your head. The root of the pirate problem was never <planet>'s fleet. You've destroyed many pirate ships since you took to the stars, and they just keep coming. Truly, what keeps the piracy going is <planet>'s culture, a culture built on the backs of its slaves and on the payrolls of its pirate cartel bosses in their private palaces. In this moment of clarity, you see a way to eliminate the pirate threat once and for all, but it will mean breaking Tomek's orders and taking the matter into your own hands`
+		choice
+			`	(Follow Tomek's drunken orders and declare <planet> to be the newest member of the Free Worlds.)`
+			decline
+			`	(Liberate <planet>'s slaves, wipe out the cartel palaces, and destroy the pirates' infrastructure.)`
+				`	Hovering over the pirate spaceport, you demand that the pirates immediately free their slaves, abandon their ships on the ground, and evacuate their spaceport. At first, they seem to comply. Then you get a frantic radio call from a surviving rebel ship. A massive pirate armada just jumped in, and another is launching from hidden hangers all over the planet! A gravelly voice chuckles over the radio. "If you want our ships, come and get them!"`
+				`	Alarms scream. You look at your tactical display and see a massive salvo of missiles coming from the largest pirate fleet you've ever heard of. The local defense fleet you already defeated was just the vanguard for this armada. Half the pirates and marauders in the galactic south must be in the system, and it looks as if they've brought some friends along.`
```

Comment:
Good suggestion. I've changed it.

As for Bladewood's comment: I've tried to provide options for the character to be both more and less confrontational in his dealings with JJ and Freya, and later with the Senate. How would you recommend I revise?

The one area where the player gets angry is when accepting the new Senate Hearing mission when, after JJ is confronted with the freed slaves, he turns around and accuses the player of becoming a pirate himself. Is this what you're talking about? I could tone it down, or maybe have Freya step in.